blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -68,6 +68,10 @@ home.pipelineslist.header.name=Name
 home.pipelineslist.header.pullrequests=PR
 home.pipelineslist.row.failing={0} failing
 home.pipelineslist.row.passing={0} passing
+home.placeholder.linkhref=/create-pipeline
```

Comment:
will this really be localised? I don't think it will be for the time being. We dont have l10n of routes at the moment anyway